web/thesauruses/nim/operators.json

```diff
@@ -0,0 +1,237 @@
+      "code": "*"
+    },
+    "multiplication_assignment": {
+      "name": "Multiplication and assignment operator",
+      "code": "*="
+    },
+    "division": {
+      "name": "Division operator",
+      "code": "/"
+    },
+    "division_assignment": {
+      "name": "Division and assignment operator",
+      "code": "/="
+    },
+    "modulus": {
+      "name": "Modulus (remainder) operator",
+      "code": "mod"
+    },
+    "modulus_assignment": {
+      "name": "Modulus and assignment operator",
+      "code": "",
+      "not-implemented": "true"
+    },
+    "increment": {
+      "name": "Increment (add 1) operator",
+      "code": "inc"
+    },
+    "decrement": {
+      "name": "Decrement (subtract 1) operator",
+      "code": "dec"
+    },
+    "exponential": {
+      "name": "Exponential operator",
+      "code": "^"
+    },
+    "factorial": {
+      "name": "Factorial operator",
+      "code": "fac"
+    },
+    "absolute_value": {
+      "name": "Absolute value operator",
+      "code": "abs"
+    },
+    "percentage": {
+      "name": "Percentage operator",
+      "code": ""
+    },
+    "equal_to": {
+      "name": "Equality operator",
+      "code": "=="
+    },
+    "not_equal_to": {
+      "name": "Not equal to operator",
+      "code": "!="
+    },
+    "less_than": {
+      "name": "Less than operator",
+      "code": "<"
+    },
+    "less_than_or_equal_to": {
+      "name": "Less than or equal to operator",
+      "code": "<="
+    },
+    "greater_than": {
+      "name": "Greater than operator",
+      "code": ">"
+    },
+    "greater_than_or_equal_to": {
+      "name": "Greater than or equal to operator",
+      "code": ">="
+    },
+    "null_coalescing": {
+      "name": "Null coalescing operator",
+      "code": "?.",
+      "comment": "'not null' annotation allows null-free programming"
+    },
+    "is": {
+      "name": "Is operator",
+      "code": "is"
+    },
+    "is_not": {
+      "name": "Is not operator",
+      "code": "isnot"
+    },
+    "and": {
+      "name": "Logical AND operator",
+      "code": "and"
+    },
+    "and_assignment": {
+      "name": "Logical AND assignment operator",
+      "code": "",
+      "not-implemented": "true"
+    },
+    "or": {
+      "name": "Logical OR operator",
+      "code": "or"
+    },
+    "or_assignment": {
+      "name": "Logical OR assignment operator",
+      "code": "",
+      "not-implemented": "true"
+    },
+    "not": {
+      "name": "Logical NOT operator",
+      "code": "not"
+    },
+    "not_assignment": {
+      "name": "Logical NOT assignment operator",
+      "code": "",
+      "not-implemented": "true"
+    },
+    "xor": {
+      "name": "Logical XOR operator",
+      "code": "xor",
+      "not-implemented": "true"
+    },
+    "xor_assignment": {
+      "name": "Logical XOR assignment operator",
+      "code": "",
+      "not-implemented": "true"
+    },
+    "xnor": {
+      "name": "Logical XNOR operator",
+      "code": ""
```

Comment:
Does this exist or is it `not-implemented` too? I'm guessing maybe it's not implemented but you forgot the line?